File: ModelCS/AppConfig.cs
using System;
using System.Collections.Generic;
using System.Data.Entity;

namespace OOPRecords.Model
{
    public static class AppConfig
    {
        public static Type[] Services()
        {
            return new[] {
                    typeof(StudentRepository)
                };
        }

        public static IDictionary<string, Type> MainMenus()
        {
            return new Dictionary<string, Type>()
            {
                ["Students"] = typeof(StudentRepository)
            };
        }

        public static DbContext CreateDbContext()
        {
            return new DatabaseContext("OOPRecords");
        }
    }
}

File: ModelVB/AppConfig.vb
Imports System.Data.Entity

Public Module AppConfig
    Function Services() As Type()
        Return {GetType(StudentRepository)}
    End Function

    Function MainMenus() As IDictionary(Of String, Type)
        Return New Dictionary(Of String, Type)() From
             {{"Students", GetType(StudentRepository)}}
    End Function

    Function CreateDbContext() As DbContext
        Return New DatabaseContext("OOPRecords")
    End Function
End Module
